### ISSUE TITLE
compatibility with the circle pad pro on old 3ds?

### ISSUE THREAD

**Issue by TonyDorf** [Roberto-Nessy/OoT3D_Standalone_Free_Cam#8]
hello friend. it's really amazing what you have done. it's a dream to play oot like this. i know it's not easy at all. but if you could really in the future make the oot free camera mod compatible with old 3ds with circle pad pro. it would be amazing. my new 3ds died... and really, to be able to play like this on the old 3ds. it would be spectacular. thanks.

**Comment by RUANRUI1995** [comment]
This feature is really needed, hope it can be added

**Comment by N0RDICST0RM** [comment]
I'd super appreciate this too.

**Comment by Gbobjr** [comment]
Same here, Would Love this.

**Comment by EthanMac1915** [comment]
Would lovd to see this happen too if possible!!

**Comment by tristangnl** [comment]
Hi, I’m glad to announce that I managed to get this working with the Circle Pad Pro.
Here’s the fork:
https://github.com/tristangnl/OoT3D_Standalone_Free_Cam_CPP

**Comment by EthanMac1915** [comment]
Amazing!! Ill try this out as soon as I get home. Incredible effort!

**Comment by EthanMac1915** [comment]
Sorry, I finally installed this to try it out and its amazing!! Hoping to get a full playthrough of the game done now that I can play it in comfort like this

**Comment by N0RDICST0RM** [comment]
Thanks so much for your efforts, is there any possibility of shoulder button functionality being added at any point? I tried to brute force it with rehid, but both can't work at once

**Comment by EthanMac1915** [comment]
Oh yeah i forgot im using a testing version of rehid from Kartik that works with CPP so atp I have full control of all the buttons

**Comment by N0RDICST0RM** [comment]
I'm using the same testing version actually, but the most recent build he sent me can't simultaneously remap cpp buttons and work with this mod unfortunately

**Comment by EthanMac1915** [comment]
oh interesting! Hopefully they put out an updated version of rehid with all the compatibility then. im glad it'd gotten so far already!

**Comment by N0RDICST0RM** [comment]
Yeah, it's ridiculously impressive. The ability to rebind to the touch screen lets you use ZL and ZR for the I and II items, so it really improves the experience. If we could have that alongside full camera control, that would be amazing
